pulsar-package-manager/src/main/java/org/apache/pulsar/packages/manager/exception/PackageMetaAlreadyExistsException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pulsar.packages.manager.exception;
+
+/**
+ * Thrown package metadata already existAsync exceptions.
+ */
+public class PackageMetaAlreadyExistsException extends PackageException {
```

Comment:
why do we have two `AlreadyExistsException`? Does it really happen that the package exists but the metadata doesn't exist?

Reply:
Because the metadata and package are two files, delete a package that needs to delete metadata as well. Delete package and metadata are two operations and I can not promise after deleting package the metadata will delete successfully. 
If one of the delete operations was failed and the user ignores the error and uploads the same package again, the metadata or the package may be already existent.
Do you have some advice?